config/config.go

```diff
@@ -95,6 +95,7 @@ type Component struct {
 type Module struct {
 	ID           string          `mapstructure:"id"`
 	Network      network.Network `mapstructure:"network" validate:"required"`
+	Type         string          `mapstructure:"type"`
```

Comment:
Is it necessary to append a `Type` field in config file? It will affect all existing nodes and the whole deployment scripts, should be more compatible.

Reply:
No, I've removed the unnecessary field and related code. With the separate component structure for federated and decentralized systems, there's no need to differentiate the type in the Module